README.md

```diff
@@ -58,14 +58,32 @@ The backend is built by the Maven `mvn clean install` command, and yields a
 "fat" executable jar containing all dependencies.
 
 The backend server is started as a standard Java executable
-using `java -jar BOM-bar-<version>.jar`.
+using `java -jar BOM-base-<version>.jar`.
 
 Some useful command line parameters are:
 
 - `--server.port=9090` changes the http port to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
 
+### Scancode  
+Install scancode-toolkit to use bombase with scancode option.
+Find how to install and configure based on OS at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install
```

Comment:
Markdown allows you to create a more beautiful link like this: `Follow these [installation instructions](https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install).`